README.md

```diff
@@ -1182,6 +1187,8 @@ Add createDate to each inserted to collection document
 
 Add updateDate to each updated or replaces document
 
+⚠️ Do no work properly with [`findOneAndUpsert()`](#findoneandupsert).
```

Comment:
```suggestion
⚠️ Do not work properly with [`findOneAndUpsert()`](#findoneandupsert).
```

Reply:
It would be good to pointing out that in case of using `findOneAndUpsert` in replace mode, field `createDate` will be rewrote every time